examples/list_cluster_creators.go

```diff
@@ -41,11 +41,14 @@ func main() {
 		os.Exit(1)
 	}
 
+	transport := sdk.NewLoggingTransport(logger)
+
```

Comment:
Put this in a separate example, and explain in the comments of that example what the example is about.

Reply:
Fixed